src/components/AngerNoodle/AngerNoodle.jsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Board from './Board';
+import MessageBox from './MessageBox';
+import Score from './Score';
+import DifficultyControls from './DifficultyControls';
+import './AngerNoodle.css';
+import computeNextFrame from './helperFunctions/gameLogic';
+import {
+  updateBoardData,
+  createDefaultBoard,
+  makeRandomPositionIndex,
+  angryMessages
+} from './helperFunctions/data';
+import handleKeyPress from './helperFunctions/eventHandlers';
+
+const AngerNoodle = ({ angerNoodleHighScore, updateAngerNoodleHighScore }) => {
+  const defaultBoard = createDefaultBoard([5, 10]);
+  const defaultTreat = makeRandomPositionIndex(defaultBoard);
+  const defaultSnake = [
+    [10, 2],
+    [10, 3],
+    [10, 4],
+    [10, 5],
+    [10, 6],
+    [10, 7],
+    [10, 8],
+  ];
+  const [treatCoords, updateTreatCoords] = useState(defaultTreat);
+  const [portalsAllowed, updatePortalsAllowed] = useState(false);
+  const [gridSize, setGridSize] = useState(15);
```

Comment:
Since your gridSize doesn't change yet it doesn't technically need to be a piece of state